Exercise 2.5

```diff
@@ -0,0 +1,27 @@
+#Diontre Davis
+#Public Policy 890
+
+#1. Why does this code not work?
+#my_variable <- 10
+#my_varıable
+#> Error in eval(expr, envir, enclos): object 'my_varıable' not found
+#Answer: The error appeared because the instructions for this assignment were 
```

Comment:
This isn't how I answered this. The character "ı" is different from the character "i" used in the word "variable," causing an error.